app/konnect/gateway-manager/data-plane-nodes/upgrade.md

```diff
@@ -16,10 +16,64 @@ We recommend running one major version (2.x or 3.x) of a data plane node per con
 Read through the [{{site.base_gateway}} upgrade considerations](/gateway/latest/upgrade/) for the version that you're upgrading to.
 
 ## Upgrade a data plane node
-
 To upgrade a data plane node to a new version, follow these steps:
 
-1. Open {% konnect_icon runtimes %} **Gateway Manager**, choose a control plane,
+{% navtabs %}
+{% navtab Dedicated Cloud Gateways %}
+{% navtabs %}
+{% navtab Konnect UI %}
+1. Open [**Gateway Manager**](https://cloud.konghq.com/us/gateway-manager/), choose the Dedicated Cloud Gateways powered control plane,
+1. From the **Actions** drop-down menu, select **Update Cluster Config**
+1. Select the desired {{site.base_gateway}} version, and **Update Cluster**.
+
+{{site.konnect_short_name}} will upgrade the version of all the data plane nodes to the version you selected automatically. 
```

Comment:
```suggestion
{{site.konnect_short_name}} will upgrade the version of all the data plane nodes to the version you selected automatically. 

Kong performs a rolling upgrade of the fully-managed data plane nodes. This is a zero downtime upgrade because Kong synchronizes the data plane with load balancer registration and deregistration and gracefully terminates the the old data plane nodes to reduce the impact on the ongoing traffic.
```
Something like this (edited from the eng response yesterday) to explain how the upgrade happens and what Kong does to make it zero downtime.